Source/AliveLibAE/Greeter.cpp

```diff
@@ -472,8 +513,7 @@ EXPORT signed __int16 Greeter::vTakeDamage_447C20(BaseGameObject* pFrom)
         return 1;
 
     default:
-        // Seems that this can be eElectricWall_39 - maybe others ??
-        LOG_WARNING("This might be wrong - greeter default damage case, can this only be a slig ?? type is " << static_cast<int>(pFrom->field_4_typeId));
+        LOG_WARNING("This might be wrong - greeter default damage case, type is " << static_cast<int>(pFrom->field_4_typeId));
         if (static_cast<BaseAnimatedWithPhysicsGameObject*>(pFrom)->field_20_animation.field_10_frame_delay <= 0)
```

Comment:
this part can be removed now since you moved the code to an explicit case statement